contrib/rootfs-builder/get-stage3.sh

```diff
@@ -39,9 +39,9 @@ fi
 
 BASE_ARCH_URL="${BASE_ARCH_URL:-${MIRROR}releases/${BASE_ARCH}/autobuilds/}"
 LATEST=$(wget -O - "${BASE_ARCH_URL}latest-stage3.txt")
-DATE=$(echo "${LATEST}" | sed -n "s|/stage3-${STAGE3_ARCH}-[0-9]*[.]tar[.]bz2.*||p")
-ARCH_URL="${ARCH_URL:-${BASE_ARCH_URL}${DATE}/}"
-STAGE3="${STAGE3:-stage3-${STAGE3_ARCH}-${DATE}.tar.bz2}"
+DATE=$(echo "${LATEST}" | sed -n "s|/.*stage3-${STAGE3_ARCH}-systemd-[0-9]*[.]tar[.]bz2.*||p" | tail -1)
```

Comment:
Thanks for pointing out the current breakage :).

I don't think we want to use the systemd image, since we're just grabbing the BusyBox binary.  It's [currently][1] quite a bit bigger:

```
20180308T214502Z/stage3-amd64-20180308T214502Z.tar.xz 188851072
…
20180303/systemd/stage3-amd64-systemd-20180303.tar.bz2 281873162
```

I'm in the process of updating the rootfs builder for the new format, but haven't gotten through the variable compression (xz, bz2, …) yet.  I'll file a PR once I've got it working again.
[1]: http://distfiles.gentoo.org/releases/amd64/autobuilds/latest-stage3.txt

Reply:
Ok, I'll rebase my PR when yours get merged.